lib/active_merchant/billing/gateways/payu_co.rb

```diff
@@ -0,0 +1,18 @@
+module ActiveMerchant #:nodoc:
```

Comment:
This doesn't seem to add value, since it's exactly the same as the values in PayuLatamGateway. Further, what is the difference besides currencies and countries between PayU Colombia and PayU Latam in general?